splunk_connect_for_snmp_mib_server/mib_server.py

```diff
@@ -50,19 +52,45 @@ def translator():
             logger.debug(request.json)
             var_binds = request.json.get("var_binds")
             metric = request.args.get("metric")
+            return_multimetric = request.args.get("return_multimetric")
             logger.debug(f"type of var_binds: {type(var_binds)}")
             logger.debug(f"var_binds: {var_binds}")
+            logger.debug(f"return_multimetric: {return_multimetric}")
             logger.debug(f"type of metric: {str(metric)} -- metric: {metric}")
             if metric == "True":
                 # if metric is true, var_binds has just one element
                 var_bind = var_binds[0]
-                result = self._translator.format_metric_data(var_bind)
+                result = json.dumps(self._translator.format_metric_data(var_bind))
             else:
-                result = self._translator.format_trap_event(var_binds)
+                # when 'return_multimetric' variable is set up as 'True', mib server should return both metric and
```

Comment:
So it looks like MIB server can return data in 3 modes: metric, multimetric and nonmetric. Maybe we should use one arg requests instead of two. For example enum MIB_DATA_FORMAT: METRIC, MULTIMETRIC, TEXT.
I'd also rename format_trap_event to e.g. format_text_event as this now used not only in context of traps.